server/controller/userController.js

```diff
@@ -4,7 +4,26 @@ import requests from '../seedData/request';
  * @class Users
  */
 class Users {
-
+  /**
+ * @return {Object} all user requests
+ * @param {param} req
+ * @param {param} res
+ */
+  static retrieveRequests(req, res) {
+    if(requests.length === 0) {
+      return res.status(200).json({
+        status: 'fail',
+        message: 'Sorry, you have not made any request'
+      })
+    } else {
+      const myRequests = requests.map(request => Object.assign({}, request));
+      return res.status(200).json({
+        status: 'success',
+        message: 'Your requests',
```

Comment:
This message is not descriptive enough. Tell me that my requests have been obtained or gotten successfully.

Reply:
I will fix that right now